config/sync/user.role.health_condition_author_user.yml

```diff
@@ -16,10 +16,46 @@ permissions:
   - 'flag content_audit'
   - 'moderate content from draft to needs_review'
   - 'moderate content from needs_review to draft'
+  - 'revert application revisions'
```

Comment:
Do Health condition authors need edit permissions for all content types?

Reply:
Nope!